lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -394,16 +464,18 @@ def main():
         force=dict(type='bool', default=False),
         appends=dict(type='bool', default=False, aliases=['incremental']),
         ipam_driver=dict(type='str', default=None),
-        ipam_options=dict(type='dict', default={}),
+        ipam_options=dict(type='dict', default={}, removed_in_version='2.12'),
+        ipam_config=dict(type='list', elements='dict', default=[]),
+        enable_ipv6=dict(type='bool', default=None),
         internal=dict(type='bool', default=None),
         debug=dict(type='bool', default=False)
     )
 
     client = AnsibleDockerClient(
         argument_spec=argument_spec,
         supports_check_mode=True,
-        min_docker_version='1.9.0'
-        # "The docker server >= 1.9.0"
+        min_docker_version='1.10.0'
```

Comment:
For what precisely is this needed?

Reply:
see https://github.com/ansible/ansible/pull/47492#discussion_r227458601

Reply:
If you only add `ipv6_enabled` if it is set to `True`, there's no need to require 1.10.0 then.

Reply:
I don't know how the behavior is if one defines an IPv6 subnet without setting enable_ipv6 to yes with docker version < 1.10.0. There might be errors.

Reply:
And after checking docker-py 1.9.0 docs, I saw the newly introduced parameter `internal` requires Docker 1.10.0 for externally isolated overlay networks: https://docker-py.readthedocs.io/en/1.9.0/networks.html#network-creation

Reply:
Ok, then let's keep it. (We can still change that later if we want, before Ansible 2.8 is released. If someone wants to invest the work...)